src/helm/benchmark/window_services/neruips_local_window_service.py

```diff
@@ -0,0 +1,27 @@
+from .local_window_service import LocalWindowService
+from .tokenizer_service import TokenizerService
+
+
+class NeuripsWindowService(LocalWindowService):
+    def __init__(self, service: TokenizerService):
+        super().__init__(service)
+
+    @property
+    def max_sequence_length(self) -> int:
+        return 2048
+
+    @property
+    def max_request_length(self) -> int:
+        return self.max_sequence_length
+
+    @property
+    def end_of_text_token(self) -> str:
+        return "<|endoftext|>"
```

Comment:
Do the special tokens need to be user-configurable?

Reply:
Yes I would imagine so, not sure what the best way of specifying that would be

Reply:
One idea is to introduce `tokenizers.yaml`:

`tokenizers.yaml`

```
tokenizers:
  - name: "efficiencychallenge/tokenizer-1"
    client_spec:
      class_name: "helm.proxy.clients.http_client.HTTPClient"
      args:
        url: "http://localhost:8000/"
        prefix_token: "<|endoftext|>"
        end_of_text_token: "<|endoftext|>"
```


`model_deployments.yaml`

```
model_deployments:
  - name: efficiencychallenge/model-1
    model_name: efficiencychallenge/model-1
    tokenizer_name: "efficiencychallenge/tokenizer-1"
    max_sequence_length: 2048
    client_spec:
      class_name: "helm.proxy.clients.http_client.HTTPClient"
      args:
        url: "http://localhost:8000/"
        do_cache: false
```

cc @percyliang any thoughts on configurable tokenizers?

Reply:
I like configurable tokenizers.

Reply:
Cool, I'll draft a PR for that.

In the meantime, I don't think we need to block this PR; we can keep these changes and clean things up later when we have configurable tokenizers.